app/workers/parse_raw_event_job.rb

```diff
@@ -28,16 +28,34 @@ def perform(raw_event_id)
         raw_event_id: re.id
       )
 
+      producer = Poseidon::Producer.new(
+        ["#{LensServer.config.kafka.host}:#{LensServer.config.kafka.port}"],
+        'lens_bg_producer'
+      )
+
+      base_hash = {
+        application: application.id,
+        scenario: scenario.events_hash,
+        conroller: meta[:controller],
+        action: meta[:action]
+      }
+
+      messages = []
       details.each_with_index do |row, index|
-        new_request.events.create(
-          event_type: row[:type],
-          content: row[:content],
-          duration: row[:duration],
-          started_at: row[:start],
-          finished_at: row[:finish],
-          position: index
+        hash = base_hash.merge(
+          {
+            timestamp: Time.at(row[:start]).to_s(:iso8601),
```

Comment:
Do not use `Time.at` without zone. Use one of `Time.zone.at`, `Time.at.current`, `Time.at.in_time_zone`, `Time.at.utc`, `Time.at.getlocal`, `Time.at.iso8601`, `Time.at.jisx0301`, `Time.at.rfc3339`, `Time.at.to_i`, `Time.at.to_f` instead.